.github/workflows/ci.yml

```diff
@@ -15,10 +15,22 @@ jobs:
           make lint
   run:
     runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.python == '2.7' }}
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest]
-        python: ['2.7', '3.7', '3.8', '3.9', '3.10', '3.11', '3.12-dev', 'pypy-3.8']
+        python:
+        - '2.7'
+        - '3.7'
+        - '3.8'
+        - '3.9'
+        - '3.10'
+        - '3.11'
+        - '3.12'
+        - '3.13-dev'
```

Comment:
In a future iteration, I recommend changing this to simply `'3.13'` and adding [the `allow-prereleases` flag](https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#allow-pre-releases) to the `setup-python` action.